### ISSUE TITLE
/stop /start command skipping songs

### ISSUE THREAD

**Issue by linushh** [BinaryBuddiesHQ/SunBotJs#11]
Right now you can use the command /stop to stop a playlist or queue. That will remove the current playing song from the queue and then you can use /start to resume the queue in it's previous state.

Example:
Currently playing: 
song 0

In queue:
Song 1
Song 2
Song 3

When you use /stop command Song 0 is stoppped and removed and Song 1 will replace song 0 in the index. There will now be in the queue:
Song 0
Song 1
Song 2 (song 3 is moved to 2, song 2 is moved to 1 etc)

If you skip till you only have one song left and then try to skip again you get an error, which you shouldn't.

TL:DR: Probably a issue with my logic using the shift() function in /stop, /start and /skip.

PS: I think this is how the problem came about, but im a silly meathead and my memory is foggy.

**Comment by malformedpeach** [comment]
i've pushed a commit containing a solution comment in the skip.js file, have at it!
